statichealthbars/sugarcube/sugarcube_statichealthbars_twee.txt

```diff
@@ -0,0 +1,12 @@
+:: StoryTitle
+Static Healthbars for SugarCube
+
+:: Start
+<<set $health to 80>>
+
+Show a healthbar using a Progress element:
+<<print '<progress value="' + $health + '" max="100"></progress>' >>
```

Comment:
@videlais 
You should be using a [HTML Attribute - Attribute Directive](http://www.motoslave.net/sugarcube/2/docs/#markup-html-attribute-directive) instead of a `<<print>>` macro to inject the value of the story variable.

eg. Replace the two `<<print>>` macro calls with
```
<progress @value="$health" max="100"></progress>
<meter @value="$health" min="0" max="100"></meter>
```